package.json

```diff
@@ -186,26 +186,55 @@
 		"@wordpress/is-shallow-equal": "^4.2.1",
 		"@wordpress/jest-preset-default": "^7.1.3",
 		"@wordpress/url": "^3.3.1",
-		"autoprefixer": "^10.2.5",
-		"babel-jest": "^27.3.1",
 		"babel-loader": "^8.2.3",
 		"backstopjs": "5.1.0",
 		"browserslist": "^4.16.0",
 		"calypso": "workspace:^",
 		"calypso-codemods": "workspace:^",
-		"capture-website": "^1.0.0",
 		"chai": "^4.3.4",
 		"chalk": "^4.1.2",
-		"check-node-version": "^4.0.2",
-		"chroma-js": "^2.1.2",
 		"concurrently": "^5.1.0",
 		"config": "^3.3.6",
-		"core-js-bundle": "^3.6.4",
 		"css-loader": "^6.5.1",
 		"debug": "^4.3.3",
+		"i18n-calypso": "workspace:^",
+		"i18n-calypso-cli": "workspace:^",
+		"jest": "^27.3.1",
```

Comment:
I'm a bit confused as to how we decided which dependency to go where. Let's take testing for example: how come a testing library ends up being a dependency for production? Perhaps I don't understand the purpose or the criteria for re-organization - if that's the case, I'd love to get some further clarification.

Reply:
That's a fair question :) The reason in this case is that the `git diff` is miscommunicating what happened. Every package under `dependencies` is already under `dependencies` in production. I didn't move or add anything to dependencies.

This PR isn't (yet) exhaustive. E.g. I did not fully check every single dependency. I went through and moved most of the lowest hanging fruit -- packages referenced very rarely, or which are very clearly dev dependencies like eslint.

It can be tricky to tell why some of the packages even need to be in the top-level package.json. I guess there are some top-level "projects" (./bin and most things under ./tests/ outside of tests/e2e) which use this package.json file? So I was trying to be cautious and avoid moving anything which might have a big impact like webpack.

However, I don't think I need to be that cautious. In reality there is no scenario where `yarn install` wouldn't install all of these packages if they are in dev dependencies, so it should still be safe to move them. I'll go ahead and move more of the clear dev dependencies like typescript, webpack, jest, etc.